components/Buttons/StyledDeleteButton.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export default function StyledDeleteButton({ children, ...props }) {
+  return <StyledButton {...props}>{children}</StyledButton>;
+}
+
+export const StyledButton = styled.button`
+  color: var(--delete-color);
+  background-color: var(--secondary-color-background);
+  padding: 0.5em;
+  border: 1px solid var(--delete-color);
+  border-radius: 0.3em;
+  cursor: pointer;
+  &:hover {
+    color: var(--secondary-color-background);
+    background-color: var(--delete-color);
+    transition: 500ms ease-in-out;
+  }
+`;
```

Comment:
_Just an advise_ 
Spreading `...props` can be incredibly convenient, but when you explicitly define props, it's clear which props the component expects and uses. This can serve as a form of documentation and makes the code easier to understand at a glance. 
Further more passing unnecessary props can lead to unnecessary re-renders if those props change, potentially affecting performance.